test/Microsoft.Health.Fhir.Shared.Tests.Integration/Persistence/SqlServerFhirStorageTestHelper.cs

```diff
@@ -206,18 +217,32 @@ public async Task DeleteDatabase(string databaseName, CancellationToken cancella
                 return;
             }
 
-            SqlConnection.ClearAllPools();
-
-            await _dbSetupRetryPolicy.ExecuteAsync(async () =>
```

Comment:
Why do we not want to retry the cleanup?

Reply:
I was mainly trying to find where all the time is spent for integration tests. I think we wait the max amount of time on retries to delete the database at the end of every fixture.
Strangely enough, when watching the resource group the databases all seemed to _eventually_ drop, perhaps there's something async going on under the covers on the sql side.

On top of this, we delete the resource group at the end of the test which should cover anything that failed, and CI has a task to remove orphaned test databases.